server/flask_server.py

```diff
@@ -0,0 +1,90 @@
+'''
+ RasPiにWebサーバを設置する 
+ @auther  desty505 
+'''
+import os
+import csv
+import re
+from flask import Flask, url_for, request
+
+app = Flask(__name__)
+
+# GETリクエストに対する操作
+@app.route('/', methods=["GET"])
+def read():
+    """関数の簡単な説明.
+    関数の詳細
+    read()は"http://"サーバIPアドレス":8000/"にGETリクエストがされたときに実行される
+    
+    Returns:
+        str型:  サーバが持つデータ
+    """
```

Comment:
形式としては、こんな感じでお願いします。
文章は案です。

```suggestion
    """GETリクエストに対しcsvファイルの内容を返す.

    "http://サーバIPアドレス:8000/"にGETリクエストされたときに実行される
    
    Returns:
        str:  サーバが持つcsvファイルの内容
    """
```

他の関数も同様。